src/rules/javascript/content-scripts-file-absent.js

```diff
@@ -0,0 +1,48 @@
+import { isBrowserNamespace, normalizePath } from 'utils';
+import { CONTENT_SCRIPT_NOT_FOUND, CONTENT_SCRIPT_EMPTY } from 'messages/javascript';
+
+export default {
+  create(context) {
+    const existingFiles = context.settings.existingFiles || {};
+    return {
+      MemberExpression(node) {
+        if (node.object.object && isBrowserNamespace(node.object.object.name)) {
+          const namespace = node.object.property.name;
+          const property = node.property.name;
+          // namespace should be tabs function should be executeScript
```

Comment:
Nit, as a convention all the inline comments should have the first letter capitalized and ends with a dot.